packages/web-components/src/components/va-accordion/va-accordion-item.scss

```diff
@@ -60,75 +58,10 @@
   display: none;
 }
 
-/* duplicate print media block needed to override above definition */
-@media print {
-  :host(:not([open])) #content,
-  :host([open='false']) #content {
-    display: block;
-  }
-}
-
-h1,
-h2,
-h3,
-h4,
-h5,
-h6 {
-  margin: 0;
-}
-
 button {
-  cursor: pointer;
-  width: 100%;
-  border: 0;
-  text-align: left;
   font-family: var(--font-source-sans);
-  font-size: 1rem;
-  font-weight: 700;
+  font-size: 1.06rem;
   line-height: 1.15;
```

Comment:
This button `line-height` is applying from normalize.css in the USWDS component but we don't have that loading directly in the component-library (it loads in vets-website globally.